### PR TITLE
Feature what to bring list

### PR DESCRIPTION
**Pull request by luukaass**

### CODE REVIEW

File: app/Http/Controllers/EditWhatToBringListController.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        PermissionFactory::createShowEventExtended()->check($id);
+
+        return view('what-to-bring-list-edit')->with(['eventID' => $id, 'items' => $this->getItems($id)]);
+    }
+
+    public function store(Request $request)
+    {
+
+        // check for permission to edit the event
+        //PermissionFactory::createEditEvent()->check();
```
Comment (tarjmp) [inline app/Http/Controllers/EditWhatToBringListController.php]: Permission check still commented out

File: app/Http/Controllers/EditWhatToBringListController.php [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        $this->validateInput($data);
+
+        $id = $data['eventID'];
+
+        // Update the event with passed data
+        $item = new Item();
+        $item->event_id = $id;
+        $item->name = $data['name'];
+        $item->amount = $data['amount'];
+        if (isset($data['user'])) $item->user_id = Auth::user()->id;
```
Comment (tarjmp) [inline app/Http/Controllers/EditWhatToBringListController.php]: Pretty print?

File: app/Http/Controllers/EditWhatToBringListController.php (hunk 1 above)
Comment (luukaass) [reply app/Http/Controllers/EditWhatToBringListController.php]: Fixed 1ad922170872005874c956c9a2d4b417760bf900

File: app/Http/Controllers/EditWhatToBringListController.php (hunk 2 above)
Comment (luukaass) [reply app/Http/Controllers/EditWhatToBringListController.php]: Was intently done by me, but adjusted with 43863f25c3b9739bf871aa048dc89ca73363fb62

**Pull request merged**